Client/Frontend/Widgets/AlternateSimpleHighlightCell.swift

```diff
@@ -0,0 +1,199 @@
+    }()
+
+    private lazy var domainLabel: UILabel = {
+        let descriptionLabel = UILabel()
+        descriptionLabel.font = DynamicFontHelper.defaultHelper.DeviceFontDescriptionActivityStream
+        descriptionLabel.textColor = AlternateSimpleHighlightCellUX.DescriptionLabelColor
+        descriptionLabel.textAlignment = .Left
+        descriptionLabel.numberOfLines = 1
+        return descriptionLabel
+    }()
+
+    private lazy var timeStamp: UILabel = {
+        let timeStamp = UILabel()
+        timeStamp.font = DynamicFontHelper.defaultHelper.DeviceFontSmallActivityStream
+        timeStamp.textColor = AlternateSimpleHighlightCellUX.TimestampColor
+        timeStamp.textAlignment = .Right
+        return timeStamp
+    }()
+
+    private lazy var siteImageView: UIImageView = {
+        let siteImageView = UIImageView()
+        siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
+        siteImageView.clipsToBounds = true
+        siteImageView.contentMode = UIViewContentMode.Center
+        siteImageView.layer.cornerRadius = AlternateSimpleHighlightCellUX.CornerRadius
+        siteImageView.layer.borderColor = AlternateSimpleHighlightCellUX.BorderColor.CGColor
+        siteImageView.layer.borderWidth = AlternateSimpleHighlightCellUX.BorderWidth
+        siteImageView.layer.masksToBounds = true
+        return siteImageView
+    }()
+
+    private lazy var statusIcon: UIImageView = {
+        let statusIcon = UIImageView()
+        statusIcon.contentMode = UIViewContentMode.ScaleAspectFit
+        statusIcon.clipsToBounds = true
+        statusIcon.layer.cornerRadius = AlternateSimpleHighlightCellUX.CornerRadius
+        return statusIcon
+    }()
+
+    private lazy var selectedOverlay: UIView = {
+        let selectedOverlay = UIView()
+        selectedOverlay.backgroundColor = AlternateSimpleHighlightCellUX.SelectedOverlayColor
+        selectedOverlay.hidden = true
+        return selectedOverlay
+    }()
+
+    override var selected: Bool {
+        didSet {
+            self.selectedOverlay.hidden = !selected
+        }
+    }
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        layer.shouldRasterize = true
+        layer.rasterizationScale = UIScreen.mainScreen().scale
+
+        isAccessibilityElement = true
+
+        contentView.addSubview(siteImageView)
+        contentView.addSubview(descriptionLabel)
+        contentView.addSubview(selectedOverlay)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(timeStamp)
+        contentView.addSubview(statusIcon)
+        contentView.addSubview(domainLabel)
+
+        siteImageView.snp_remakeConstraints { make in
+            make.top.equalTo(contentView).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.bottom.equalTo(contentView).offset(-AlternateSimpleHighlightCellUX.CellTopBottomOffset).priorityLow()
```

Comment:
Instead of making this a low priority, it might also work with a `greaterThanOrEqualTo` constraint. I assume this is the constraint between the bottom of the image and the cell? Also, if the text beside it is set up with constraints which forces the height of the cell we might not even need this constraint at all and it can just be pinned to the top/leading.